### PR TITLE
roachtest: fix tpchvec to parse EXPLAIN ANALYZE (DEBUG) correctly

### PR DESCRIPTION
**Pull request by yuzefovich**
The output of `EXPLAIN ANALYZE (DEBUG)` has been changed which broke the
assumption that the "direct link" line is the last one. This commit
fixes the parsing code to be more flexible when searching for that line.

Release note: None

### CODE REVIEW

File: pkg/cmd/roachtest/tpchvec.go [hunk 1]
```diff
@@ -351,26 +351,30 @@ func (p *tpchVecPerfTest) postTestRunHook(
...
+					// we will curl below.
+					directLinkPrefix := "Direct link: "
+					var line, url string
 					for rows.Next() {
-						if err = rows.Scan(&lastLine); err != nil {
+						if err = rows.Scan(&line); err != nil {
 							t.Fatal(err)
 						}
+						if strings.HasPrefix(line, directLinkPrefix) {
+							url = line[len(directLinkPrefix):]
```
Comment (jordanlewis) [inline pkg/cmd/roachtest/tpchvec.go:368]: can't we `break` here?

**Review by jordanlewis — APPROVED** [review]

**Review by yuzefovich — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/51658)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @helenmhe and @jordanlewis)

___
*[pkg/cmd/roachtest/tpchvec.go, line 368 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/51658#-MCmjX42DDZjd2Wvw3lL-r1-368:-MCmkjSA-jWoZymryYHH:b-rru5zj) ([raw file](https://github.com/cockroachdb/cockroach/blob/aa84ff0464f9a290f3fa1f912c53865607fbb867/pkg/cmd/roachtest/tpchvec.go#L368)):*
<details><summary><i>Previously, jordanlewis (Jordan Lewis) wrote…</i></summary><blockquote>

can't we `break` here?
</blockquote></details>

We could, but I remember an issue when `rows` wasn't fully consumed and had a useful error that would be returned with `rows.Close` but only if `rows` has been exhausted. This whole code section is a temporary thing that I'm hoping to remove some time soon, so I wouldn't pay too close of attention to it.

**Comment by yuzefovich** [comment]
TFTR!

bors r+

**Comment by yuzefovich** [comment]
I guess I can add a `break`, no big deal.

bors r-

**Pull request merged**